ibm/service/vpc/resource_ibm_is_vpc_default_routing_table.go

```diff
@@ -0,0 +1,363 @@
+// Copyright IBM Corp. 2017, 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const ()
+
+func ResourceIBMISVPCDefaultRoutingTable() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMISVPCDefaultRoutingTableCreate,
+		Read:     resourceIBMISVPCDefaultRoutingTableRead,
+		Update:   resourceIBMISVPCDefaultRoutingTableUpdate,
+		Delete:   resourceIBMISVPCDefaultRoutingTableDelete,
+		Exists:   resourceIBMISVPCDefaultRoutingTableExists,
+		Importer: &schema.ResourceImporter{},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			rtVpcID: {
```

Comment:
it's computed and force new, hence getting the below error
<img width="957" alt="Screenshot 2023-02-17 at 10 35 00 AM" src="https://user-images.githubusercontent.com/78336632/219553959-bfd6d398-f284-45e1-8e9b-f7008a2851a2.png">